src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Security/ReviewCodeForSqlInjectionVulnerabilitiesTests.cs

```diff
@@ -622,6 +624,40 @@ protected void Page_Load(object sender, EventArgs e)
                 GetCSharpResultAt(15, 37, 15, 52, "SqlCommand.SqlCommand(string cmdText)", "void WebForm.Page_Load(object sender, EventArgs e)", "string HttpRequest.this[string key]", "void WebForm.Page_Load(object sender, EventArgs e)"));
         }
 
+        [Fact]
+        public async Task HttpRequest_Form_Method_GenericsSink_Diagnostic()
+        {
+            var csharpTest = new VerifyCS.Test
+            {
+                ReferenceAssemblies = ReferenceAssemblies.NetFramework.Net472.Default
+                    .AddAssemblies(ImmutableArray.Create("System.Web", "System.Web.Extensions"))
+                    .AddPackages(ImmutableArray.Create(new PackageIdentity("EntityFramework", "6.4.4"))),
+                TestState =
+                {
+                    Sources = { @"
+namespace VulnerableWebApp
+{
+    using System;
+    using System.Data.Entity;
+    using System.Web.UI;
+
+    public partial class WebForm : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
+        {
+            string input = Request.Form.Get(""in"");
+            new DbContext(""connectionString"").Set<Object>().SqlQuery(input, null);
+        }
+     }
+}
+" }
```

Comment:
If you apply the previous change you need to also apply this one:

```suggestion
",
```